CMakeLists.txt

```diff
@@ -2,15 +2,16 @@ cmake_minimum_required(VERSION 3.5.1)
 project(CMaizeSuperbuild VERSION 1.0.0 LANGUAGES CXX)
 
 # This is where we will install artifacts during the build stage
-# TODO: change stage_dir to CMAKE_INSTALL_DIR
 set(stage_directory "${CMAKE_BINARY_DIR}/stage/${CMAKE_INSTALL_PREFIX}")
 
+# Tell compiler to not include debug flags
 set(CMAKE_BUILD_TYPE_INIT "Release")
 
+# Add cmake config location to paths to search
+list(APPEND CMAKE_PREFIX_PATH ${stage_directory}/lib/cmake)
```

Comment:
You should be able to leave off the `lib/cmake` part. CMake has a set of platform-specific subfolders that it looks in below the prefix thus insulating you from needing to change the path depending on the platform.

Reply:
To make this work I'd need to change CMakePublic to place its config file in INSTALL_PREFIX/INSTALL_LIBDIR/PROJECT_NAME or INSTALL_PREFIX/INSTALL_LIBDIR/PROJECT_NAME/cmake instead of INSTALL_PREFIX/INSTALL_LIBDIR/cmake.
I went ahead and tested this on a local version of CMakePublic I had. I also think it's cleaner than just having all the config files dumped into one folder.

Reply:
the config files for CMakePublic should get placed in `INSTALL_PREFIX/cmakepublic/INSTALL_LIBDIR/cmake`. The libraries should go in `INSTALL_PREFIX/cmakepublic/INSTALL_LIBDIR` and the include files should go in `INSTALL_PREFIX/cmakepublic/INSTALL_INCDIR` (or whatever the variable for includes is...). The point is there should be a directory with the name `cmakepublic` that acts as a namespace so that the files don't just get mixed in with `INSTALL_PREFIX/lib` etc. (on Linux systems INSTALL_PREFIX/lib and the like are usually reserved for the OS's package management)

Reply:
If it's not doing this already then that's an error with the CMakePublic repo and needs fixed.